src/btree.c

```diff
@@ -1006,7 +1007,7 @@ static void verify_btree_block(struct super_block *sb,
 	int hashed = 0;
 	int end_off;
 	int tot = 0;
-	int i = 0;
+	int i = 0, j = 0;
```

Comment:
One definition per line, please.  (I added it to the nascent coding style text file!)